src/federation-info-plugin/FederationInfo/FederationInfo.jsx

```diff
@@ -51,18 +57,31 @@ const FederationInfoContent = ({ federationSchemaUrl }) => {
     )
   }
 
+  console.log(servicesSchemaResize)
```

Comment:
remove

Reply:
strange that linter didn't pick this up, need to look at it, because it shouldn't allow it